### PR TITLE
⚡  Optimize the purchase flow by prefetching and preconnecting from/to ti.to

### PR DESCRIPTION
**Pull request by cramforce**
Prefetches the main doc and known sub resources whenever a doc finishes loading.

On mouseover and click of the purchase button, preconnects to ti.to and other origins that ti.to uses to flatten the connection tree.

### CODE REVIEW

File: contents/js/main.js [hunk 1]
```diff
@@ -9,10 +9,51 @@
...
+  addLink('preconnect', 'https://titoproduction.global.ssl.fastly.net');
+  addLink('preconnect', 'https://maps.googleapis.com');
+}
+
+function prefetchTito() {
+  // Only do this before the event
+  if (Date.now() > new Date('2018-07-01').getTime()) {
+    return;
+  }
+  addLink('prefetch', 'https://ti.to/jsconfeu/jsconf-eu-2018');
```
Comment (robin-drexler) [inline contents/js/main.js]: Out of curiosity: Does prefetch have a positive effect on (html) resources with max-age set to 0?
Aren't cache headers respected?

Comment (robin-drexler) [reply contents/js/main.js]: Thanks! Are you aware of any document describing that behavior? Could not find it here: https://w3c.github.io/resource-hints/#prefetch

Comment (cramforce) [reply contents/js/main.js]: Not aware, but it definitely works :)

CC @igrigorik

Comment (igrigorik) [reply contents/js/main.js]: @robin-drexler @cramforce indeed, works in Blink-based browsers.. Not 100% on other browsers. This is an un-specced corner of the platform, see: https://github.com/w3c/preload/issues/97

Comment (robin-drexler) [reply contents/js/main.js]: Thanks to you both. :)

**Pull request merged**